tests/bridges/cctp/ICctpMessageTransmitter.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
why is this interface in the tests folder?

Reply:
some more functions added this interface for test

Reply:
I changed some configuration of MessageTransmitter for test. for this, this interface have some more functions.